### PR TITLE
Fix search facets

### LINKED ISSUE #8: add facets for category and taxon to search
from #5

### PR DESCRIPTION
**Pull request by nathandunn**
Fixes #8

Will be adding categories. 

- [x] add taxon to searchAPI ?
- [x] use TaxonMap to search
- [x] create headers for category vs taxon section in facet counts and categories, as well

### CODE REVIEW

**Comment by nathandunn** [comment]
image
image
image

**Comment by nathandunn** [comment]
@DoctorBud  @jmcmurry  I think this i ready for a UI and technical review.   Screenshots above.

**Comment by jmcmurry** [comment]
looks good; is it deployed on pages somewhere for inspection of behavior?

**Comment by kshefchek** [comment]
Do we want to have the default weights and filters used for autocomplete? 
boost_q=category:genotype^-10
prefix=-OMIA

**Comment by DoctorBud** [comment]
@nathandunn can you do an `npm run ghpublish` so that we can see what it looks like at nathandunn.github.io/monarch-ui please? That let's folks see the UI without necessarily building it. Of course, if you have overlapping PRs, then it will fail to be useful (Don't do that!).

**Comment by nathandunn** [comment]
@DoctorBud  that is awesome.  FYI @jmcmurry 

@kshefchek  I don't know.  Should those be passed into the API call, or are they there by default?

https://nathandunn.github.io/monarch-ui/

**Comment by nathandunn** [comment]
@kshefchek  fixes in

**Comment by kshefchek** [comment]
+1!

File: src/views/Search.vue [hunk 1]
```diff
@@ -18,51 +18,138 @@
...
-              >
-                <span v-html="row.item.highlight"/>
-              </template>
-            </b-table>
+      <div class="row">
+        <div
+          class="container col-md-3"
+        >
+          <div
+            v-if="categoryFilters.length>0 || taxonFilters.length>0"
```
Comment (DoctorBud) [inline src/views/Search.vue]: Not sure why the ESLint isn't picking this up, but you should have spaces around binary operators like `>`.

File: src/views/Search.vue [hunk 2]
```diff
@@ -18,51 +18,138 @@
...
+            class="b-row card"
+            style="margin-top: 20px">
+            <div class="card-header">
+              Taxa
+            </div>
+            <div class="card-body">
+              <ul class="showFacetLinks  col-md-12">
+                <li v-for="(value,propertyName) of facetTaxons ">
+                  <a
+                    href="javascript:"
```
Comment (DoctorBud) [inline src/views/Search.vue]: What is the `href="javascript:"` about? Seems like you could just remove this.

File: src/views/Search.vue [hunk 3]
```diff
@@ -18,51 +18,138 @@
...
+            class="b-row card"
+            style="margin-top: 20px">
+            <div class="card-header">
+              Categories
+            </div>
+            <div class="card-body">
+              <ul class="showFacetLinks col-md-12">
+                <li v-for="(value,propertyName) of facetCategories">
+                  <a
+                    href="javascript:"
```
Comment (DoctorBud) [inline src/views/Search.vue]: `href="javascript:"` makes no sense to me

**Comment by DoctorBud** [comment]
+1

**Pull request merged**